MAINTAINING.md

```diff
@@ -96,3 +96,401 @@ write effective API documentation, such as setting up the RST files and docstrin
+The release cycle for major and minor releases is regular so these can wait, and if it is a bugfix, we are free to make another patch release quickly.
+Non-blocking issues with no associated PR within a day of the release
+should be pushed for further release.
+
+## Release process for Qiskit
+
+This section contains the human-intervention parts of the release process for pushing a release and hosting it on PyPI.
+
+The technical process of building, testing and deploying the package is automated.
+However, the **release manager** should manually follow these steps:
+
+ 1. [Check the milestone state](#1-check-the-milestone-state)
+ 2. [Audit `Changelog:*` labels](#2-audit-changelog-labels)
+ 3. [Creating, submitting, and merging a PR to handle release-specific changes as last commit on the milestone](#3-submit-a-prepare-xyz-release-pr)
+ 4. [Tag the commit from step 3 to trigger the release automation process](#4-tag-the-prepare-xyz-release-commit)
+ 5. [Make post-release changes in the repository](#5-post-release-actions)
+
+The release process is largely the same for all versions.
+However, for convenience, let's put names to two main scenarios for a release:
+
+ * a **_first_ release**: If [`stable/x.y` does not exist in upstream repository](https://github.com/Qiskit/qiskit/branches), you are probably preparing for the first release candidate of a major or minor release, and you are in this scenario.
+   Examples of *first* releases are: `0.43.0`, `1.0.0rc1`, and `1.3.0rc1`.
+ * a **_follow-up_ release**: If [`stable/x.y` exists in the upstream repository](https://github.com/Qiskit/qiskit/branches), this release is some form of follow-up, probably because there is a previous release candidate or you are doing a patch release.
+   Examples of *_follow-up_* releases are: `0.43.3`, `1.2.1`, and `1.3.0rc2`.
+
+> [!NOTE]
+> This section assumes that the Qiskit-owned GitHub remote is called `upstream` in your git configuration.
+
+
+### 1. Check the milestone state
+
+Verify that the milestone is in a suitable place to release:
+
+ - Set the due date for an estimated time for the release, if not set already (for example, in patch release cases).
+ - Check for missing items in the milestone. For example, [search for open PRs against stable branches](https://github.com/Qiskit/qiskit/pulls?q=is%3Apr+is%3Aopen+-base%3Amain) and ensure they are labeled with the upcoming release milestone.
+ - The day before the release: 
+   * all the blocking issues/PR should be merged the day before the release.
+   * if it is an rc release, you can leave non-critical bug fixes open, as they could land later, before the final release.
+   * non-blocking issues/PR can left open and consider bumping to the next release milestone later. 
+
+### 2. Audit `Changelog:*` labels
+
+> [!NOTE]
+> In this section, `(x.y.z)-1` refers to the **the previous version tag**, since the tool needs to consider changes _since_ the version supplied.
```

Comment:
```suggestion
> In this section, `(x.y.z)-1` refers to **the previous version tag**, since the tool needs to consider changes _since_ the version supplied.
```